### DISCUSSION TITLE
What do we think about the discussions board?

### DISCUSSION THREAD

**Discussion by Bento007** [chanzuckerberg/single-cell-data-portal#2213] [General]
The discussion board has been enabled. What kind of things should we discuss here?

**Comment by Alex2975** [comment]
Dear authors, I was trying to following the instruction in "CZ CELLxGENE Discover" to install a local deployment environment. But I do not have permission to view the confluence link from the following step. My installation failed and not able to get the docker image due to the lack of AWS ID, I believe. The error message is "ERROR: no such image: .dkr.ecr.us-west-2.amazonaws.com/corpora-upload-success: invalid reference format". Could you please help me out? Thank you very much. 

Set up your machine to be able to work with AWS using the instructions [here](https://czi.atlassian.net/wiki/spaces/DC/pages/332892073/Getting+started+with+AWS). Please ensure to follow the step 3 AWS CLI access instructions all the way to the bottom so that you are also set up for SSH access. When you run the final command that requires the team's infra repo, use single-cell-infra.

**Reply by MaximilianLombardo** [reply]
Hi Alex!

Thanks for the question! As a follow-up, may I ask what you are trying to accomplish exactly?

This repo is for the [CELLxGENE data portal](https://cellxgene.cziscience.com/) and while the code is definitely open source, it is a repo that is more meant for our internal developers (hence the permissions errors that you encountered when following up on some of the links). The consequence of this is that the docs in this repo are currently not in a state where an external user/developer could stand up their own version of the portal (or at least not without significant effort) and it is not currently one of our goals to enable this for users (due to both the complexity of that task and the need to prioritize against our other development efforts). 

That being said, if you let me know of what your goals are, then I'd be happy to point towards alternative solutions.

**Reply by Alex2975** [reply]
Dear MaximilianLombardo,

Thank you so much for getting back to me so quickly! I was planning to install this software on my laptop to organize single cell studies, especially the visualization for the harmonized single cell studies. But sounds like it is very challenging for external users. If there are other mature options you could point me to, I would appreciate that very much.

**Reply by MaximilianLombardo** [reply]
> to organize single cell studies

What type of organization would you be doing beyond the exploration of harmonized studies (more follow up on that below)?

> especially the visualization for the harmonized single cell studies

Would you be generating integrated representations of public/private single cell datasets and then visualizing the integrated object?

> But sounds like it is very challenging for external users. If there are other mature options you could point me to, I would appreciate that very much.

Yes, it is not an advisable path, but based on your answers to the above questions I'm hoping we can find a suitable direction for you!

**Reply by Alex2975** [reply]
Dear MaximilianLombardo,

Thank you. 

Yes, we will generate integrated representations of public single cell datasets and want to visualize the integrated objects. We want to have separate tabs (Collections, Datasets, Gene Expression) to organize, visualize our single cell datasets, just like what your data portal will present. We are especially interested to have the "Gene Expression" tab like visualization, if there is a good option.
